src/app/admin/admin/admin.component.html

```diff
@@ -0,0 +1,18 @@
+<div fxLayout="column" fxLayoutAlign="center center" style="padding-top: 35px;" style="border-color: violet; border-style: solid;">
+    <div fxLayout="row" style="border-color:red; border-style: solid; width: 800px;" >
+        <div fxFlex=50 fxLayoutAlign="start center" style="border-color:black; border-style: solid;">
+
```

Comment:
remove empty lines throughout the file

Reply:
As mentioned early this is a component in development so please don't consider this for review.

Reply:
well i thought the development was complete and hence added reviewers? I would not expect any changes/ files in the prq if they are not needed or still in development